virttest/qemu_vm.py

```diff
@@ -3941,6 +3943,26 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
                                % (key, state, s))
                         raise exceptions.TestError(msg)
 
+            # set migrate_parameter in source and target before migration
+            # starts
+            if migrate_parameters:
+                logging.info("Set migrate parameters before migration")
+                for parameter, value in migrate_parameters.items():
+                    self.monitor.set_migrate_parameter(parameter, value)
+                    s = self.monitor.get_migrate_parameter(parameter)
+                    if s != value:
+                        msg = ("Migrate parameter '%s' should be '%s', "
+                               "but actual result is '%s' on source guest"
+                               % (parameter, value, s))
+                        raise exceptions.TestError(msg)
+                    clone.monitor.set_migrate_parameter(parameter, value)
```

Comment:
As far as I know, not all parameters need to be set on destination qemu, actually during our test most parameters only being set on source qemu.

Reply:
@vivianQizhu correct, I will handle it. Thank you reviewing.